scss/partials/_back_to_dashboard_btn.scss

```diff
@@ -0,0 +1,19 @@
+div.back-to-dashboard-row {
+  margin-top: 5px;
+  padding: 20px;
+
+  button.back-to-dashboard {
+    width: 200px;
+    height: 30px;
+    border: solid 1px rgba(78, 90, 107, 0.2);
+    font-weight: 700;
+    font-size: 12px;
+    line-height: 1.5;
+    letter-spacing: 1px;
+    color: rgba(78,90,107, 0.5);
+    margin: 0px;
+    padding: 0px;
+    background-color: white;
+    outline: none;
+  }
+}
```

Comment:
These styles button styles are reused across the app. I added a few styles in the `improve redirect logic` PR for this type of button that are easier to reuse: https://github.com/open-company/open-company-web/pull/147/files#diff-36a224998480a3e2239228e57a8a1e76R7

Also consider using the basscss classes `mt1` - `mt4` for margin top and the `p1` - `p4` classes for uniform paddings. 

http://www.basscss.com/#basscss-margin
http://www.basscss.com/#basscss-padding